test/libsolidity/util/TestFileParser.h

```diff
@@ -98,13 +98,22 @@ namespace soltest
  */
 struct ABIType
 {
-	enum Type {
+	enum Type
+	{
 		UnsignedDec,
 		SignedDec,
 		Failure,
 		None
 	};
+
+	enum Align
```

Comment:
@chriseth This does not necessarily need to be added with this PR, since `AlignLeft` is only used in https://github.com/ethereum/solidity/pull/6036. But I felt that it's not worth the effort splitting it out again after cherry-picking.

Reply:
ok!